data2bids.m

```diff
@@ -2030,7 +2085,17 @@
       sampling_frequency =cell(length(name), 1);% sampling frequency of nirs channels are not required
     end
   end
-  tab = table(name, type, units, sampling_frequency, source, detector, wavelength);
+  % add motion specific fields (empty columns will be removed in a later
+  % step)
+  component=cell(length(name), 1); reference_space=cell(length(name),1);
+  if isfield(hdr, 'chansource');
+    for i=1:length(name)
+      source{i}=hdr.chansource(i);
+      component{i}=hdr.chancomponent(i);
+      reference_space{i}=hdr.chanrefspace(i);
```

Comment:
how would this work with a dataset that is converted from C3D to BIDS? Or a dataset that was already read in MATLAB memory, e.g. using `ft_preprocessing`?